src/transaction.ts

```diff
@@ -1560,12 +1561,97 @@ export class Transaction extends Dml {
             resp.commitTimestamp as DateStruct
           );
         }
+        err = Transaction.decorateCommitError(err as ServiceError, mutations);
```

Comment:
@skuruppu This adds additional information to the error for the specific case that the application tries to insert an array of objects into a JSON column.

Reply:
Perfect, thanks for doing this. If you could also please add some unit tests for this, that would be awesome.

Reply:
Good point, added.

Reply:
Thanks a lot for adding the tests.